docs/proposals/The proposal about building a high frequency Sedna based end to end use case in ModelBox format.md

```diff
@@ -0,0 +1,86 @@
+[toc]
+
+# The proposal about building a high frequency Sedna based end to end use case in ModelBox format
+
+## 1. Background
+
+> Sedna is an edge cloud collaborative AI project at KubeEdge SIG AI. Benefiting from the edge-cloud collaboration capabilities provided by KubeEdge, Sedna can realize cross-edge-cloud collaborative training and collaborative reasoning capabilities, and Sedna supports a variety of popular AI frameworks. **Sedna can simply enable edge-cloud synergy for existing training and inference scripts**, bringing the benefits of lower costs, improved model performance, and data privacy. **The goal of ModelBox is to solve the programming complexity of AI developers when developing AI applications**, reduce the development difficulty of AI applications, and hand over complex data processing to ModelBox. Developers mainly focus on the business logic itself, not the software details. While improving the efficiency of AI inference development, it ensures the performance, reliability, security and other attributes of the software, **so as to better enable Sedna developers to manage and use the Sedna framework**.
+
+## 2. Motivation
+
+### Goals
+
+**The ModelBox application encapsulates the Sedna edge cloud collaborative training function module and the fully functional edge cloud collaborative AI application case (incremental training function)**.
+
+## 3. Proposal
+
+- **Use the expandable development function of ModelBox, use C++ to develop functional units, package Sedna Library in ModelBox, and call Sedna API to realize relevant functions.**
+- **Carry out flow chart unit development on ModelBox, call the encapsulated incremental learning API of Sedna library, upload data to the cloud side through the incremental learning training model, upload data to the cloud and train data respectively, call the cloud testing training API of Sedna Library in ModelBox, train in the cloud, send the training results back to the side, compare the accuracy with the side training data, and realize the incremental training helmet detection sample**
+
+
+
+## 4. Design Details
+
+### Flow chart of ModelBox integrated Sedna edge-cloud collaboration function
+
+![ModelBox intergrated Sedna flow chart](images\ModelBox intergrated Sedna flow chart.png)
```

Comment:
As mentioned by @back1860, an architecture graph is needed in the sedna perspective.